---
layout: post
title: Soft and Hard Homogenization Protocol
date: '2019-06-03'
categories: Protocols
tags: soft homogenization, hard homogenization, DNA extraction, RNA extraction, protocol
projects: Holobiont Integration
---

# Soft and Hard Homogenization Protocol
Goal: To extract DNA and RNA from coral fragments and to separate host, symbiont, and microbiome cells.  
End Analysis: Transcriptomics on host RNA, DNA methylation on host DNA, ITS2 amplicon sequencing on symbiont DNA, 16s amplicon sequencing on microbiome DNA.

"Soft homogenization" will include DNA and RNA from the host for transcriptomics and DNA methylation analysis. "Hard homogenization" will include the DNA from the symbionts and microbiome for ITS2 and 16s amplification.

### Protocol Preparation

**Reagents and supplies**
- RNAse-free water
- 100% ethanol, ACS grade or better
- 10mM Tris HCl pH 8.0 made with RNase-free water
- Lysis tubes with beads (specs??)

**Equipment**
- Rocking oven that can be set at 55 &deg;C
- RNAse away and a designated RNAse free space
- Tabletop and larger centrifuges for 1.5 mL and 50 mL tubes capable of 12,000xg

### Fragment Preparation

Sterilizing working area:
I. Rub down clippers with:
  1. 10% bleach solution
  2. DI water
  3. 90% Ethanol
  4. RNAse free water

II. Wipe down benchtop with RNAse away and kim wipes. Spray gloves with RNAse away and wipe with kim wipes.

Take fragments one at a time out of the freezer and sterilize clippers in between every fragment. These steps are time sensitive to prevent the coral fragments and freezer from thawing. Do not rush, but be efficient.

Clipping fragment:
1. Label the side of a 1.5 mL falcon tube with the coral fragment's sample ID number, date of collection, "hard", your initials, and today's date. Label the cap of the falcon tube with the coral ID number.
2. Label a 1.5 mL falcon tube with the coral fragment's sample ID number, date of collection, "hard", your initials, and today's date. Label the cap of the falcon tube with the coral ID number.
3. Add 1000 uL of RNA/DNA shield, 100 uL of Protenase K digestion buffer, and 50 uL of Protenase K to the 1.5 mL falcon tube labeled "hard".
4. Remove desired fragment from -80 &deg;C freezer. Close the freezer door.
5. Using sterilized clippers, clip off only 1/2 of the coral fragment into 4-5 pieces and place into the 1.5 mL falcon tube labeled "hard".

*Repeat above steps 1-5 for each fragment before moving on.*

6. Vortex in (insert name of vortex sample holder) for 2 minutes. Leave the settings on and on max power.
7. After 2 minutes, check to see if the tissue is coming off of the skeleton. *2 minutes might need to be extended.*
8. Remove supernatant from the "hard" falcon tube and place in the 1.5 falcon tube labeled "soft". This supernatant will become the sample for "soft homogenization". The remaining chunks of fragment will become the sample for "hard homogenization".
9. Add X uL of RNA/DNA shield each "hard" falcon tube.
10. Add X beads (insert name of beads) to each "hard" falcon tube.
11. Place the "hard" falcon tubes in the Qiagen tissue bead beater for X minutes.
12. Gently remove beads from the falcon tube with sterilized forceps (following the sterilization protocol above).
13. Place the "soft" and "hard" falcon tubes in Thermomixer for incubation for X minutes at X &deg;C.

## Zymo Duet RNA DNA Extractions

Modified from the [Zymo protocol](https://github.com/kevinhwong1/KevinHWong_Notebook/blob/master/images/Zymo_quick-dna-rna_miniprep_plus_kit.pdf).

**DNA Extraction**
1. Set up yellow DNA spin columns and collection tubes, label appropriately
2. Warm elution liquids to 70 degrees C (10mM Tris HCl pH. 8.0 and RNase free water)
3. Add equal volume (345µl) DNA/RNA lysis buffer to each sample tube
4. Finger flick to mix tubes
5. Add 700µl (total volume) of sample gently to the yellow DNA spin column
6. Centrifuge at 16,000 rcf (g) for 30 seconds
7. **Important** Save the flow through from this step: transfer to a new 1.5mL tube labeled for RNA
8. Add 400µl DNA/RNA Prep Buffer gently to the yellow DNA spin columns
9. Centrifuge at 16,000 rcf (g) for 30 seconds
10. Discard flow through (Zymo kit waste)
11. Add 700µl DNA/RNA Wash Buffer gently to the yellow DNA spin columns
12. Centrifuge at 16,000 rcf (g) for 30 seconds
13. Discard flow through (Zymo kit waste)
14. Add 400µl DNA/RNA Wash Buffer genetly to the yellow DNA spin columns
15. Centrifuge at 16,000 rcf (g) for **2 minutes**
16. Discard flow through (Zymo kit waste)
17. Transfer yellow columns to new 1.5mL microcentrifuge tubes
18. Add 50µl warmed 10mM Tris HCl to each yellow DNA column by dripping slowly directly on the filer
19. Incubate at room temp for 5 minutes
20. Centrifuge at 16,000 rcf (g) for 30 seconds
21. Repeat steps 18-20 for a final elution volume of 100µl
22. Label tubes, store at 4 degrees C if quantifying the same day or the next, if waiting longer store in -20

**RNA Extraction**
*Can do concurrently with DNA Extraction after DNA Extraction Step 7*
1. Add equal volume (700µl) 100% EtOH to the 1.5mL tubes labeled for RNA containing the original yellow column flow through
2. Vortex and spin down to mix
3. Add 700µl of that liquid to the green RNA spin columns
4. Centrifuge at 16,000 rcf (g) for 30 seconds
5. Discard flow through (Zymo kit waste)
6. Add 700µl to the green RNA spin columns (the rest from the 1.5mL RNA tubes)
7. Centrifuge at 16,000 rcf (g) for 30 seconds
    - Get DNase I from freezer
8. Discard flow through (Zymo kit waste)
9. Add 400µl DNA/RNA Wash Buffer gently to each green RNA column
10. Centrifuge at 16,000 rcf (g) for 30 seconds
11. Discard flow through (Zymo kit waste)
12. Make DNase I treatment master mix:
    - 75µl DNA Digestion buffer x # of samples
    - 5µl DNase I x # of samples
13. Add 80µl DNase I treatment master mix directly to the filter of the green RNA columns
14. Incubate at room temp for 15 minutes
15. Add 400µl DNA/RNA Prep Buffer gently to each column
16. Centrifuge at 16,000 rcf (g) for 30 seconds
17. Discard flow through (Zymo kit waste)
18. Add 700µl DNA/RNA Wash Buffer gently to the yellow DNA spin columns
19. Centrifuge at 16,000 rcf (g) for 30 seconds
20. Discard flow through (Zymo kit waste)
21. Add 400µl DNA/RNA Wash Buffer genetly to the yellow DNA spin columns
22. Centrifuge at 16,000 rcf (g) for **2 minutes**
23. Discard flow through (Zymo kit waste)
24. Transfer green columns to new 1.5mL microcentrifuge tubes
25. Add 50µl warmed DNase/RNase free water to each green RNA column by dripping slowly directly on the filer
26. Incubate at room temp for 5 minutes
27. Centrifuge at 16,000 rcf (g) for 30 seconds
28. Repeat steps 25-27 for a final elution volume of 100µl
29. Label 1.5mL tubes on ice afterwards, and aliquot 5µl into PCR strip tubes to save for Qubit and Tape Station to avoid freeze-thaw of your stock sample
30. Store all tubes in the -80

## Aliquoting for further analysis
From the "soft homogenization" falcon tubes:  
1. Aliquot X uL into a microcentrifuge tube labeled with the coral fragment ID, collection date, extraction date, and "DNA methylation".
2. Aliquot X uL into a microcentrifuge tube labeled with the coral fragment ID, collection date, extraction date, and "16s".
3. Aliquot X uL into a microcentrifuge tube labeled with the coral fragment ID, collection date, extraction date, and "ITS2".


## Clean-up
1. Rinse beads use for hard homogenization in ethanol and DI water.
2. Wipe down RNA free area with RNase away and kimwipes.
3. Throw away all tips and restock tip boxes if necessary.

## Testing Quantity and Quality
To test RNA and DNA quantity: [Qubit](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/Qubit-Protocol/)  
To test RNA quality: [TapeStation](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/RNA-TapeStation-Protocol/)  
To test DNA quality: [Gel Electrophoresis](https://meschedl.github.io/MESPutnam_Open_Lab_Notebook/Gel-Protocol/)
